providers/org/plos/harvester.py

```diff
@@ -0,0 +1,46 @@
+from furl import furl
+from lxml import etree
+
+from project.settings import PLOS_API_KEY
+
+from share import Harvester
+
+
+class PLOSHarvester(Harvester):
+
+    url = 'http://api.plos.org/search'
+    MAX_ROWS_PER_REQUEST = 999
+
+    def do_harvest(self, start_date, end_date):
+        start_date = start_date.date()
+        end_date = end_date.date()
+
+        return self.fetch_rows(furl(self.url).set(query_params={
+            'q': 'publication_date:[{}T00:00:00Z TO {}T23:59:59Z]'.format(start_date.isoformat(), end_date.isoformat()),
```

Comment:
Don't call `.date()` on `start_date` and `end_date` if you are using the time part of it.
Look at the OAI harvester class for date formatting tips